aws/aws-util/src/main/java/io/atleon/aws/util/AwsConfig.java

```diff
@@ -76,6 +80,8 @@ public static AwsCredentialsProvider loadCredentialsProvider(Map<String, ?> conf
         switch (type) {
             case CREDENTIALS_PROVIDER_TYPE_DEFAULT:
                 return DefaultCredentialsProvider.create();
+            case CREDENTIALS_PROVIDER_WEB_IDENTITY_TOKEN_FILE:
+                return WebIdentityTokenFileCredentialsProvider.create();
```

Comment:
Also super nitty, but let's put this case after `CREDENTIALS_PROVIDER_TYPE_STATIC` so that it's ordered alphabetically